cli/server.go

```diff
@@ -15,21 +15,64 @@ import (
 
 	"cdr.dev/slog"
 	"cdr.dev/slog/sloggers/sloghuman"
+	"github.com/coder/code-marketplace/extensionsign"
 
 	"github.com/coder/code-marketplace/api"
 	"github.com/coder/code-marketplace/database"
 	"github.com/coder/code-marketplace/storage"
 )
 
+func serverFlags() (addFlags func(cmd *cobra.Command), opts *storage.Options) {
+	opts = &storage.Options{}
+	var sign bool
+	return func(cmd *cobra.Command) {
+		cmd.Flags().StringVar(&opts.ExtDir, "extensions-dir", "", "The path to extensions.")
+		cmd.Flags().StringVar(&opts.Artifactory, "artifactory", "", "Artifactory server URL.")
+		cmd.Flags().StringVar(&opts.Repo, "repo", "", "Artifactory repository.")
+		cmd.Flags().DurationVar(&opts.ListCacheDuration, "list-cache-duration", time.Minute, "The duration of the extension cache.")
+		cmd.Flags().BoolVar(&sign, "sign", false, "Sign extensions.")
```

Comment:
You are right it does not make the most sense, but it is hidden. The `sign` flag will be removed though, replaced by something that will parse a key or cert.

See the flags in my other PR: https://github.com/coder/code-marketplace/blob/449f77c63cf60655b1cfd22e71cb3ad3b85488fa/cli/server.go#L35-L37

I do have a flag to save the signature to disk. Makes it easier to debug

Reply:
I'm going to leave `sign` for now. I intend to fix it before being unhidden, anyone using it will be broken when I change things.


When I fix it, I'll make the flags make sense.